src/cirrus/management/commands/manage.py

```diff
@@ -221,11 +246,15 @@ def process(deployment):
 @click.argument(
     "lambda-name",
 )
+@pass_session
 @pass_deployment
-def invoke_lambda(deployment, lambda_name):
+def invoke_lambda(deployment: Deployment, session: boto3.session, lambda_name: str):
```

Comment:
That is a module, `Session` is the type.
```suggestion
def invoke_lambda(deployment: Deployment, session: boto3.session.Session, lambda_name: str):
```